Jenkinsfile_CNP

```diff
@@ -7,13 +7,15 @@ import uk.gov.hmcts.contino.HealthChecker
 
 properties([
         parameters([
-                string(name: 'PRODUCT_NAME', defaultValue: 'ccd', description: ''),
+                choice(name: 'PRODUCT_NAME', choices: 'probate\nccd', description: 'product name like probate,ccd, div.'),
+                choice(name: 'DATA_BASE_NAME', choices: 'probatemandb\nccd-data-store-api', description: 'DB to which logstash attaced (to read keys from valut)'),
```

Comment:
Lets rather lookup $DATA_BASE_NAME based on the value of $PRODUCT_NAME, but not a blocker